gradle.properties

```diff
@@ -11,6 +11,12 @@ android.nonTransitiveRClass=true
 #MPP
 kotlin.mpp.enableCInteropCommonization=true
 kotlin.mpp.androidSourceSetLayoutVersion=2
+kotlin.mpp.enableGranularSourceSetsMetadata=true
+kotlin.native.enableDependencyPropagation=false
+
+kotlin.native.cacheKind=none
+kotlin.native.binary.memoryModel=experimental
+kotlin.native.binary.freezing=disabled
```

Comment:
Why did you add these configurations?

Reply:
We're using kotlin 1.7.10 in this project, and those settings are required to keep Koin working properly.